tern/analyze/default/container/single_layer.py

```diff
@@ -148,3 +148,42 @@ def analyze_first_layer(image_obj, master_list, redo):
     for p in image_obj.layers[0].packages:
         master_list.append(p)
     return shell
+
+
+def analyze_live(image_layer):
+    # set up a notice origin for the given image layer
+    origin_image_layer = 'Layer {}'.format(image_layer.layer_index)
+    # check if the layer is empty
+    if com.is_empty_layer(image_layer):
```

Comment:
Will have to check if we can actually use all the common and default_common functions in the "live" use case. I will need to take a look further before I can approve this PR.